Sources/Build/llbuild.swift

```diff
@@ -308,20 +307,23 @@ public struct LLBuildManifestGenerator {
         })
 
         // For Clang, the target requires all command outputs.
-        var buildTarget = Target(name: target.target.llbuildTargetName)            
+        var buildTarget = Target(name: target.target.getLLBuildTargetName(config: plan.buildParameters.configuration.dirname))
         buildTarget.outputs.insert(contentsOf: commands.flatMap({ $0.tool.outputs }))
         buildTarget.cmds += commands
         return buildTarget
     }
 }
 
 extension ResolvedTarget {
-    public var llbuildTargetName: String {
-        return "\(name).module"
+    public func getCommandName(config: String) -> String {
+       return "C." + getLLBuildTargetName(config: config)
     }
-
-    var commandName: String {
-        return "C.\(llbuildTargetName)"
+    
+    public func getLLBuildTargetName(config: String) -> String {
+        if config == "release" {
+            return "\(name)-release.module"
+        }
+        return "\(name)-debug.module"
```

Comment:
This can just be `return "\(name)-\(config).module"

Reply:
done